link/perf_event.go

```diff
@@ -66,11 +74,9 @@ type perfEvent struct {
 	// ID of the trace event read from tracefs. Valid IDs are non-zero.
 	tracefsID uint64
 
-	// True for kretprobes/uretprobes.
-	ret bool
+	eventType perfEventType
```

Comment:
We can use `typ perfEventType` instead to avoid stuttering. (currently, this reads as `perfEvent.eventType`)